consensus/service/src/config/mod.rs

```diff
@@ -1,22 +1,24 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
+#![deny(missing_docs)]
```

Comment:
nit: this usually goes in `lib.rs`, but it's a lot of work to do that for all of consensus service, so i see if you want to punt on that